inst/include/dust/distr/normal.hpp

```diff
@@ -7,26 +7,33 @@ namespace dust {
 namespace distr {
 
 template <typename real_t>
-inline real_t box_muller(rng_state_t<real_t>& rng_state) {
+HOSTDEVICE inline real_t box_muller(rng_state_t<real_t>& rng_state) {
   // This function implements the Box-Muller transform:
   // http://en.wikipedia.org/wiki/Box%E2%80%93Muller_transform#Basic_form
   // Do not send a really small number to log().
+#ifdef __NVC__
+  // TODO: Assumes float for now - use DBL_EPSILON if real_t == double
+  const real_t epsilon = FLT_EPSILON;
+  const real_t two_pi = 2 * M_PI;
```

Comment:
This could be constexpr with a CUDA compiler flag, but minimal effort to recompute so I think this is fine